src/main/java/org/zone/commands/structure/region/flags/damage/to/player/attack/ZoneFlagDamageAttackSetEnabledCommand.java

```diff
@@ -1,4 +1,4 @@
-package org.zone.commands.structure.region.flags.damage.attack;
+package org.zone.commands.structure.region.flags.damage.to.player.attack;
```

Comment:
Not a fan of `to` either revert it or have `towards` or some other word

Reply:
Towardd